Makd.mak

```diff
@@ -751,7 +751,7 @@ fasttest: $(fasttest)
 d2conv: $O/d2conv.stamp
 
 $O/d2conv.stamp: $(D1TO2FIX_DIRS)
-	$Vfind $(D1TO2FIX_DIRS) -type f -regex '^.+\.d$$' > $@
+	$V$(call find,$(D1TO2FIX_DIRS),-type f -regex '^.+\.d$$' > $@
```

Comment:
Missing closing `)` here by the look of it?

Reply:
Note, I tried fixing this by adding a closing `)` before the `>`, and I found myself getting a `Permission denied` error on `submodules/ocean/test/collectd/main.d` when `make d2conv` was run.

Reply:
Yeah, I think this call is actually not really replaceable. I will remove this commit from the PR. Try now please.

Reply:
Yea, `d2conv` now works and this now works with both D1 and D2 builds.

Although D2 builds seem to generate some weird non-ASCII character in console output:

```
allunittests.xml^�:
```

Reply:
Not related to Makd: https://github.com/sociomantic-tsunami/ocean/pull/245